numpy/distutils/misc_util.py

```diff
@@ -2382,84 +2362,58 @@ def show():
-                   * ``define_macros``: preprocessor macros used by
-                     ``distutils.setup``
-                   * ``baseline``: minimum CPU features required
-                   * ``found``: dispatched features supported in the system
-                   * ``not found``: dispatched features that are not supported
-                     in the system
-
-                2. NumPy BLAS/LAPACK Installation Notes
-
-                   Installing a numpy wheel (``pip install numpy`` or force it
-                   via ``pip install numpy --only-binary :numpy: numpy``) includes
-                   an OpenBLAS implementation of the BLAS and LAPACK linear algebra
-                   APIs. In this case, ``library_dirs`` reports the original build
-                   time configuration as compiled with gcc/gfortran; at run time
-                   the OpenBLAS library is in
-                   ``site-packages/numpy.libs/`` (linux), or
-                   ``site-packages/numpy/.dylibs/`` (macOS), or
-                   ``site-packages/numpy/.libs/`` (windows).
-
-                   Installing numpy from source
-                   (``pip install numpy --no-binary numpy``) searches for BLAS and
-                   LAPACK dynamic link libraries at build time as influenced by
-                   environment variables NPY_BLAS_LIBS, NPY_CBLAS_LIBS, and
-                   NPY_LAPACK_LIBS; or NPY_BLAS_ORDER and NPY_LAPACK_ORDER;
-                   or the optional file ``~/.numpy-site.cfg``.
-                   NumPy remembers those locations and expects to load the same
-                   libraries at run-time.
-                   In NumPy 1.21+ on macOS, 'accelerate' (Apple's Accelerate BLAS
-                   library) is in the default build-time search order after
-                   'openblas'.
+                1. Information is derived with the help of `threadpoolctl`
+                   library.
+                2. In case `threadpoolctl` is not installed, a message is
+                   displayed with help on how to install it.
+                3. SIMD related information is derived from `__cpu_features__`,
+                   `__cpu_baseline__` and `__cpu_dispatch__`
 
                 Examples
                 --------
                 >>> import numpy as np
                 >>> np.show_config()
-                blas_opt_info:
-                    language = c
-                    define_macros = [('HAVE_CBLAS', None)]
-                    libraries = ['openblas', 'openblas']
-                    library_dirs = ['/usr/local/lib']
+                [{'architecture': 'Zen',
+                'filepath': '/usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so',
+                'internal_api': 'openblas',
+                'num_threads': 12,
+                'prefix': 'libopenblas',
+                'threading_layer': 'pthreads',
+                'user_api': 'blas',
+                'version': '0.3.20'}]
+
```

Comment:
Where is the example of the SIMD information?

Reply:
Didn't want to clutter the docstring, but yeah makes sense to add it as it'll always be displayed

Reply:
Done via [49ed995](https://github.com/numpy/numpy/pull/21468/commits/49ed9952bffcf55fa4d298e05aee3d18138fe2a3)